src/applications/ivc-champva/10-7959C/chapters/applicantInformation/personalInformation.js

```diff
@@ -16,10 +15,6 @@ export default {
   uiSchema: {
     ...titleUI(PAGE_TITLE),
     applicantName: fullNameMiddleInitialUI,
```

Comment:
Note: `fullNameUI`, which is consumed in this definition, contains this validation by default. There is no need to duplicate it.